api/models/Indicator.js

```diff
@@ -7,90 +7,91 @@
 var bcrypt = require('bcrypt');
 module.exports = {
 
-  //
-  // Rename Indicator
-  // Year
-  // And permison read only and public only
-  //
-  // Standard input Criteria
-  // Create Criteria
-  //
-  autoPK: true,
-  attributes: {
-    code: {
-      type: 'string',
-      unique: true
-    },
-    name: {
-      type: 'string',
-      required: true
-    },
-    valueType:{
-      enum:['yes/no','numeric','other']
-    },
-    public:{
-      enum:['Public','Staff Only','Not Sure']
-    },
-    subgroup: {
-      model: 'SubGroup'
-    },
-    goal1:{
-      type:'integer',
-      enum:[0,1,2,3]
-    },
-    goal2:{
-      type:'integer',
-      enum:[0,1,2,3]
-    },
-    goal3:{
-      type:'integer',
-      enum:[0,1,2,3]
-    },
-    goal4:{
-      type:'integer',
-      enum:[0,1,2,3]
-    },
-    goal5:{
-      type:'integer',
-      enum:[0,1,2,3]
+    //
+    // Rename Indicator
+    // Year
+    // And permison read only and public only
+    //
+    // Standard input Criteria
+    // Create Criteria
+    //
+    autoPK: true,
+    attributes: {
+        code: {
+            type: 'string',
+            primaryKey: true,
+            required: true
```

Comment:
code is already implemented here but never used in FE , ı changed this because this field must be primary key